package.json

```diff
@@ -12,19 +12,20 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
     "ethers": "^5.7.0",
-    "next": "12.2.5",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-reveal": "^1.2.2",
+    "next": "^13.5.4",
+    "react": "^18.2.0",
+    "react-awesome-reveal": "^4.2.5",
```

Comment:
Updated next and had to update some other dependencies to get rid of some errors